Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```
@@ -170,43 +176,57 @@ script:
     - contextPath: File.SSDeep
       description: SSDeep hash of the file.
       type: string
-  - arguments:
-    - description: Commit description.
-      name: description
-    description: Commits a configuration to the Palo Alto firewall or Panorama, but
-      does not validate if the commit was successful. Committing to Panorama does
-      not push the configuration to the firewalls. To push the configuration, run
-      the panorama-push-to-device-group command.
-    name: panorama-commit
+    description: Gets the predefined threats list from a Firewall or Panorama and
+      stores it as a JSON file in the context.
+  - name: panorama-commit
+    arguments:
+    - name: description
+      description: Commit description.
+    - name: use_polling
+      auto: PREDEFINED
+      predefined:
+      - "false"
+      - "true"
+      description: Whether to use polling.
+      defaultValue: "false"
+    - name: timeout
+      description: Timeout (in seconds) for polling.
+    - name: interval_in_seconds
+      description: Interval (in seconds) to check each polling.
```

let's add a defaultvalue
```suggestion 
      description: Interval (in seconds) to check each polling.
      defaultValue: '30'
```